### PR TITLE
mount modules  for auto load ip6tables moudles

### PR DESCRIPTION
**Pull request by wangyd1988**
#### What type of this PR
Examples of user facing changes:
- bugs
- I have encountered a situation where the ipv6 kernel could not be loaded. We solved the problem by mounting moudles to CNI

### CODE REVIEW

**Comment by zhangzujian** [comment]
The following files also need to be updated:

```txt
yamls/kube-ovn.yaml
yamls/kube-ovn-ipv6.yaml
yamls/kube-ovn-dual-stack.yaml
```

File: yamls/kube-ovn-dual-stack.yaml [hunk 1]
```diff
@@ -222,6 +225,9 @@ spec:
       nodeSelector:
         kubernetes.io/os: "linux"
       volumes:
+        - name: host-modules
+          hostPath:
+          path: /lib/modules
```
Comment (zhangzujian) [inline yamls/kube-ovn-dual-stack.yaml:230]: incorrect indent.

File: yamls/kube-ovn-ipv6.yaml [hunk 2]
```diff
@@ -202,6 +205,9 @@ spec:
       nodeSelector:
         kubernetes.io/os: "linux"
       volumes:
+        - name: host-modules
+          hostPath:
+          path: /lib/modules
```
Comment (zhangzujian) [inline yamls/kube-ovn-ipv6.yaml:210]: same

File: yamls/kube-ovn.yaml [hunk 3]
```diff
@@ -228,6 +231,9 @@ spec:
       nodeSelector:
         kubernetes.io/os: "linux"
       volumes:
+        - name: host-modules
+          hostPath:
+          path: /lib/modules
```
Comment (zhangzujian) [inline yamls/kube-ovn.yaml:236]: same

Comment (wangyd1988) [reply yamls/kube-ovn.yaml:236]: sorry

**Comment by wangyd1988** [comment]
@zhangzujian can you review  again？

**Review by zhangzujian — APPROVED** [review]

**Pull request merged**